eZ/Bundle/EzPublishCoreBundle/Resources/views/default/content/embed.html.twig

```diff
@@ -1,7 +1,10 @@
 {% set content_name=ez_content_name(content) %}
 
+{% set parameters = {
+    "isEmbed": true
+} %}
 {% if location is defined %}
-    <h3><a href="{{ ez_path(location) }}">{{ content_name }}</a></h3>
+    <h3><a href="{{ ez_path(location, parameters) }}">{{ content_name }}</a></h3>
 {% else %}
-    <h3><a href="{{ ez_path(content) }}">{{ content_name }}</a></h3>
+    <h3><a href="{{ ez_path(content, parameters) }}">{{ content_name }}</a></h3>
```

Comment:
Shouldn't this rather skip generating link if user does not have access?


As in:
- The link will always go to full view, which implies user needs to have full access
- So rather then doing the checks here in the router, the approach is maybe rather to check rights when rendering embed and pass variable to template to let it skip link if user does not have access to full view  _(if template is rendered it's already implied user has access to see it, at least embed view of it)_

Reply:
> Shouldn't this rather skip generating link if user does not have access?

@andrerom I'm not really sure right now. The links are present also in other places in the app even when the user does not have access to linked content, so that's why I decided to kinda go with this solution. Should I change it for the solution you mentioned?

Reply:
I think so, otherwise we are just pushing the issue elsewhere as the user will click on a link and get exception. But maybe @alongosz has some perspective on this as well